lib/util/js.js

```diff
@@ -60,11 +60,11 @@ module.exports = {
    * @return {Object} The target object
    */
   defineImmutable: function defineImmutable(target, values) {
-    Object.keys(values).forEach(function(key){
+    _.each(values, function(value, key) {
```

Comment:
lodash "each" will iterate over anything with a .length property, including strings. "values" here is expected to be an object, not an array or string.